editor/components/post-publish-panel/style.scss

```diff
@@ -59,3 +51,65 @@
 	color: $blue-medium-700;
 	text-decoration: underline;
 }
+
+.editor-post-publish-panel__prepublish {
+	padding: 16px;
+
+	.components-panel__body {
+		margin-left: -16px;
+		margin-right: -16px;
+		margin-bottom: -16px;
+		margin-top: 10px;
+		border: none;
+
+		.components-panel__body-toggle {
+			font-weight: normal;
+		}
+	}
+
+	.editor-post-visibility__dialog-legend {
+		display: none;
+	}
+}
+
+.post-publish-panel__postpublish .components-panel__body {
+	border-bottom: 1px solid $light-gray-500;
+	border-top: none;
+}
+
+.wp-core-ui .post-publish-panel__postpublish-buttons {
+	display: flex;
+	align-content: space-between;
+
+	> * {
+		flex-grow: 1;
+		margin-right: 10px;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+
+	.components-button {
+		text-align: center;
+	}
+
+	.components-clipboard-button {
+		width: 100%;
+	}
+}
+
+.post-publish-panel__postpublish-link-input[readonly] {
+	width: 100%;
+	border-bottom: 1px solid $light-gray-500;
+	padding: 10px;
+	border-radius: $button-style__radius-roundrect;
+	margin: 15px 0;
+	background: $white;
+  overflow: hidden;
```

Comment:
Is there a styling issue here? Looks like there's only two spaces instead of a tab - same as next line.

Reply:
yep, there's a tabs/spaces mix :)